docfx_yaml/extension.py

```diff
@@ -51,6 +51,9 @@
 from .monkeypatch import patch_docfields
 from .directives import RemarksDirective, TodoDirective
 from .nodes import remarks
+from .markdown_utils import run_sphinx_markdown
+from .markdown_utils import reformat_markdown_to_html
+from .markdown_utils import move_markdown_pages
```

Comment:
I think style says to import the module, rather than the individual methods/classes within it.

Reply:
Oops, updated. Done.